contrib/clojure-package/test/test-ndarray-random-api.clj

```diff
@@ -0,0 +1,95 @@
+(ns
```

Comment:
Do we need to have this file checked in? - it looks like it is written to during the generator test to compare contents

Reply:
Ooooops! Good catch! I will remove them